### DISCUSSION TITLE
Parsing external web server logs into security onion

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#12009] [Unsupported Versions]
Hi, 
I want to parse the external web server logs which is in file format. How can i send them to security onion.
Can you give some idea to parse the external logs in security onion 2.4.

Should i have to download filebeat! for that or i can do with elastic agent itself

**Comment by dougburks** [comment]
There are Elastic integrations for some kinds of web server logs (for example: apache). Check to see if your web server is listed at https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations.
